app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -956,42 +995,41 @@ impl Area {
 
             // === Cursor setup ===
 
-            let edit_mode = frp.input.set_editing.clone_ref();
-            let on_background_press = frp.output.on_background_press.clone_ref();
-            model.label.set_cursor_at_mouse_position <+ on_background_press.gate(&edit_mode);
-            eval edit_mode([model](edit_mode) {
-                model.label.deprecated_set_focus(edit_mode);
-                model.set_widgets_visibility(!edit_mode);
-                if *edit_mode {
-                    // Reset the code to hide non-connected port names.
-                    model.label.set_content(model.expression.borrow().code.clone());
-                    model.label.set_cursor_at_mouse_position();
-                } else {
-                    model.label.remove_all_cursors();
-                }
+            eval set_editing((is_editing) model.set_edit_mode(*is_editing));
+
+            // Disable selection mouse following right after starting edit mode to prevent it from
+            // firing on initial mouse down event as a selection start. Otherwise, a selection would
+            // be created between the initial cursor position (which takes code vis offset into
```

Comment:
What is "initial cursor position"? I know it is explained as "which takes code vis offset into account" but I still don't understand what it is and how its computed.